lib/core/src/Cardano/Wallet/Primitive/CoinSelection/MA/RoundRobin.hs

```diff
@@ -1012,8 +1022,10 @@ makeChange criteria
 
     totalInputValue :: TokenBundle
     totalInputValue = TokenBundle.add
+      (TokenBundle.add
         (F.fold inputBundles)
-        (maybe TokenBundle.empty TokenBundle.fromCoin extraCoinSource)
+        (maybe TokenBundle.empty TokenBundle.fromCoin extraCoinSource))
+      (TokenBundle.fromTokenMap mintInputs)
```

Comment:
**Suggestion:**

It might be clearer to make use of the `Monoid` instance for `TokenBundle` here:

```hs
    totalInputValue :: TokenBundle
    totalInputValue = mempty
        <> F.fold inputBundles
        <> F.foldMap TokenBundle.fromCoin extraCoinSource
        <> TokenBundle.fromTokenMap mintInputs
```

Or:

```hs
    totalInputValue :: TokenBundle
    totalInputValue =
        F.fold inputBundles
        <> F.foldMap TokenBundle.fromCoin extraCoinSource
        <> TokenBundle.fromTokenMap mintInputs
```